RELEASE/scripts/autoscend/paths/casual.ash

```diff
@@ -147,12 +147,15 @@ void acquireFamiliarsCasual()
 	}
 	hatchFamiliar($familiar[hovering sombrero]);
 	
-	//delevel enemy cheaply
-	if(!have_familiar($familiar[barrrnacle]) && item_amount($item[Barrrnacle]) == 0 && auto_mall_price($item[Barrrnacle]) < 1000 && my_meat() > 10000)
+	if(!have_familiar($familiar[Gelatinous Cubeling]) && !have_familiar($familiar[Nosy Nose]))	//if none of the common better delevel familiars
```

Comment:
it could but the other familiars in gremlins list were non standard ones that may not have the same availability as barrrnacle later, if this is also to prepare for future runs?

Reply:
Good point